### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by hhware**

### CODE REVIEW

File: format/index.md [hunk 1]
```diff
@@ -919,7 +919,8 @@ a properly generated and formatted *UUID* as described in RFC 4122
 > NOTE: In some use-cases, such as importing data from another source, it
 may be possible to use something other than a UUID that is still guaranteed
 to be globally unique. Do not use anything other than a UUID unless you are
-100% confident that the strategy you are using is indeed globally unique.
+100% confident that the strategy you are using indeed generates globally 
+unique indentifiers.
```
Comment (steveklabnik) [inline format/index.md:923]: This seems like replacing not-a-typo with a typo

File: format/index.md [hunk 2]
```diff
@@ -627,7 +627,7 @@ value for resource linkage, as described above for link objects.
 The top-level *links object* **MAY** contain `self` and `related` links,
 as described above for link objects.
 
-For example, a `GET` request a to-one relationship URL could return:
+For example, a `GET` request to a to-one relationship URL could return:
```
Comment (steveklabnik) [inline format/index.md:630]: :+1:

File: format/index.md (hunk 1 above)
Comment (hhware) [inline format/index.md:923]: @steveklabnik, sorry, not sure I understand... I thought IDs should be unique. How could uniqueness of stategy be important from the POV of JSON API?

**Comment by dgeb** [comment]
These clarifications seem fine - thanks @hhware.

**Pull request merged**